### DISCUSSION TITLE
【bug】hydro用户可以登录

### DISCUSSION THREAD

**Discussion by w1010t** [hydro-dev/Hydro#459] [Show and tell]
1、SU的菜单里点击切换账户可以随意切换到别人账户，不需要密码，我自己搭的OJ给了我和我同学SU，结果他给我号直接JC了

2、切换账户里可以选择登录hydro账户，我被我同学JC后用hydro账号给他发信息吓死他了。

3、用SU给hydro换了个密码，大家都能登进去了。。。

**Comment by w1010t** [comment]
《我太鬼才了》

**Comment by topdevst** [comment]
不给su，给个mod
